### DISCUSSION TITLE
Incorrect billing calculation for Pro.

### DISCUSSION THREAD

**Discussion by developedbygeo** [community/community#46001] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Good morning,

I have a yearly Copilot subscription, which amounts to $100. I subscribed to it in September 2022 so its renewal date is a year later. However, it incorrectly displays as a monthly recurring cost in my profile, although I am not being actually charged $100 for it every month. 

![billing-1](https://user-images.githubusercontent.com/74512500/216005720-68c41b6a-9c5e-4091-8e78-c2ed3302d4b1.JPG)

The issue though is that I want to upgrade to Pro and the incorrect Copilot calculation directly affects the Pro subscription, as you may also see in the following images. Has anyone ever encountered this? I am considering going forward with the subscription in hopes that the final payable amount will be correct, but haven't done so yet. Any feedback is much appreciated.

![billing-2](https://user-images.githubusercontent.com/74512500/216006141-203697f2-839d-447f-b527-9b285511e233.JPG)
![billing-3](https://user-images.githubusercontent.com/74512500/216006164-c158d4d3-089a-4335-b655-b36452b82909.JPG)

**Comment by jge162** [comment, marked as answer]
I have not personally had this issue. Here is a link to billing [support](https://support.github.com/contact/account?tags=hubberfy_account) Choose billing and submit scenario as you did here and they should be able to help you out.

**Reply by joneslee0918** [reply]
your answer is helpful.

**Comment by mohamadrezayousa** [comment]
1

**Comment by veloxy** [comment]
I'm also having this issue, and it also shows a one time sponsor I've already paid. (It's one time, why would that need to be included in an upgrade to pro) ![](https://i.imgur.com/NwGCzio.png)

@Github please fix your billing, it's flawed
